### ISSUE TITLE
Application crashes when converting if "Max Pal BG" is set to 1

### ISSUE THREAD

**Issue by Cuperino** [michel-iwaniec/OverlayPal#54]
# Problem

Setting "Max Pal BG" to 1 and pressing "Convert" results in application crash.

# Terminal output
```
CMPL: Writing solution to CSV file > /home/javier/.local/share/RTC/AppRun/firstpass_output.csv
AppRun: src/cpp/OverlayOptimiser.cpp:276: bool OverlayOptimiser::parseCmplSolution(const string&, std::vector<std::set<unsigned char> >&, GridLayer&, GridLayer&, Array2D<unsigned char>&, bool): Assertion `indices.size() == 3 && "Uses-palette index length mismatch"' failed.
fish: Job 1, './AppRun' terminated by signal SIGABRT (Abort)
```

OverlayPal version: 1.0
Qt version: 5.15.6
OS: Fedora Linux 36 (KDE Plasma) x86_64

**Comment by michel-iwaniec** [comment]
Hmm, this is a really weird one. It seems to be caused by a crash in QElapsedTimer and not in any OverlayPal/CMPL code. It seems it might be a bug in the Qt libraries... or perhaps a complex memory corruption issue.

Will probably need to get the full Qt sources to debug this one... and maybe even switch from Windows to Linux and use Valgrind.

**Comment by michel-iwaniec** [comment]
> Hmm, this is a really weird one. It seems to be caused by a crash in QElapsedTimer and not in any OverlayPal/CMPL code. It seems it might be a bug in the Qt libraries... or perhaps a complex memory corruption issue.
> 
> Will probably need to get the full Qt sources to debug this one... and maybe even switch from Windows to Linux and use Valgrind.

Ignore this comment - that was for a different bug. :)

But this one has been fixed in 9e515b4f7913bfc69277d3675ea8d7ff1893b6d8
